tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -127,20 +137,23 @@ let main argv = 0"""
         let mutable output = ""
         let mutable errors = ""
         let mutable cleanUp = true
-        let projectDirectory = Path.Combine(Path.GetTempPath(), "CompilerAssert", Path.GetRandomFileName())
+        let pathToTemp = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "../../../../Temp"))
```

Comment:
Why not resolve `%TEMP%`?

Reply:
Not at all, I want us to pick up the global.json in the repo.  %temp% is in the user directory, and so is nowhere near the repo